pyiron_atomistics/atomistics/structure/factory.py

```diff
@@ -332,3 +332,65 @@ def element(parent_element, new_element_name=None, spin=None, potential_file=Non
                 parent_element=parent_element, new_element=new_element_name
             )
         return periodic_table.element(new_element_name)
+
+# importing the modules required by aimsgb
+from pyiron.atomistics.structure.atoms import pyiron_to_pymatgen, pymatgen_to_pyiron
+from pymatgen import Structure, Lattice, PeriodicSite
+from aimsgb import GrainBoundary, Grain, GBInformation
+
+class GBBuilder:
+
+    def gb_info(axis, max_sigma):
+        """
+        Provides a list of possible GB structures for a given rotational axis and upto the given maximum sigma value.
+
+        Args:
+            axis : Rotational axis for the GB you want to construct (for example, axis=[1,0,0])
+            max_sigma (int) : The maximum value of sigma upto which you want to consider for your
+            GB (for example, sigma=5)
+
+        Returns:
+            A list of possible GB structures.
+        """
+        print("Possible GB structures for rotational axis",axis,"and sigma cutoff", max_sigma,"are listed in the format:\n")
```

Comment:
This can go into the docstring.

Reply:
@pmrv thank you. I will add them to the docstring.